braph2genesis/graphs/_MultilayerBUT.gen.m

```diff
@@ -88,7 +92,7 @@ In a multilayer binary undirected with fixed threaholds (BUT) graph, layers have
 value = Graph.NONNEGATIVE * ones(layernumber);
 
 %%% ¡prop!
-A (result, cell) is the cell containing the multilayer binary adjacency matrices of the multilayer binary undirected with fixed threaholds graph.
+A (result, cell) is the cell containing the within-layer binary undirected multilayer graphs of the multilayer binary undirected with fixed threaholds graph.
```

Comment:
A (result, cell) is the cell array containing the binary supra-adjacency matrix of the multilayer binary undirected with fixed thresholds (BUT) graph.